Main/CMakeLists.txt

```diff
@@ -15,8 +25,8 @@ set_source_files_properties(
     PROPERTIES HEADER_FILE_ONLY TRUE)
 
 add_executable(ivan ${IVAN_SOURCES} Resource/Ivan.rc)
-target_include_directories(ivan PUBLIC Include ../Felib/Include ../audio ../fantasyname)
-target_link_libraries(ivan FeLib FeAudio SDL2_mixer pcre xbrzscale fantasyname)
```

Comment:
I get a linker error in the MSYS2 compiler if `SDL2_mixer` is omitted. Adding it makes it work again.

Reply:
I need to fix FindSDl2.cmake. I shouldn't put the flags for SDL2_mixer into SDL2_LIBRARY and SDL2_INCLUDE_DIR.

Reply:
Should I check SDL 1.2.0 as well?

Reply:
> Should I check SDL 1.2.0 as well?

I don't know of anyone in the community who is compiling with SDL 1.2.0, so I don't think we need to support the check for that.